Babylon/commands/macro/deploy_workspace.py

```diff
@@ -231,11 +239,11 @@ def deploy_workspace(namespace: str, file_content: str, deploy_dir: pathlib.Path
                                             params=params,
                                         )
                             logger.info(f"[powerbi] report {name} successfully imported")
-        state["services"]["powerbi"]["dashboard_view"] = dashboard_view
-        state["services"]["powerbi"]["scenario_view"] = scenario_view
-        env.store_state_in_local(state)
-        if env.remote:
-            env.store_state_in_cloud(state)
+                state["services"]["powerbi"]["dashboard_view"] = dashboard_view
+                state["services"]["powerbi"]["scenario_view"] = scenario_view
+                env.store_state_in_local(state)
+                if env.remote:
+                    env.store_state_in_cloud(state)
         if workspaceCharts is not None:
```

Comment:
Ici le programme va exécuter la suite de code
```
content.get('spec').get('payload').get('webApp').get('options').get(
                'charts')['workspaceId'] = state["services"]["powerbi"]["workspace.id"]
            logger.info(
                f"[powerbi] updating workspace {state['services']['api']['workspace_id']} with all powerbi reports")
            payloadUpdated: dict = content.get("spec").get("payload")
            specUpdated = dict()
            specUpdated["payload"] = json.dumps(payloadUpdated, indent=2, ensure_ascii=True)
            workspace_svc.updateWithPayload(specUpdated)
        azure_credential = get_azure_credentials()
```
Par contre, c'est faux. Nous avons mis en commentaire la partie PowerBI, il faut ajouter à la condition :
`if workspaceCharts is not None and powerbi_section:`